packages/react-lowcode/src/codegen/generation/generators/template/template-resolver.ts

```diff
@@ -28,13 +27,11 @@ export default class TemplateResolver {
                 const inputParameterIdentifier = EntityHelper.getInputParameterIdentifier(this._entity);
 
                 //find 'useGeneratedQuery' import and replace it with use'queryName's
-
-                //TODO pascalCase function 'customer' -> 'Customers'
-                const generatedQueryName = this._entity.getName()
-                const hookName = `use${generatedQueryName.charAt(0).toUpperCase() + generatedQueryName.slice(1)}s`
+                const generatedQueryName = this._entity.getListTypeQueryName() ?? ''
+                const hookName = `use${generatedQueryName.charAt(0).toUpperCase() + generatedQueryName.slice(1)}Query`

```

Comment:
use the `camelize()`

Reply:
please move those three lines to a small function that calculates `queryHookName()` with used `camelize()`

later we will extend/generalize this functionality (it will be fine in an isolated code/function)